tests/Integration/inc/ThirdParty/Hostings/Cloudways/varnishIP.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace WP_Rocket\Tests\Integration\inc\ThirdParty\Hostings\Cloudways;
+
+use WP_Rocket\ThirdParty\Hostings\Cloudways;
+use WPMedia\PHPUnit\Integration\TestCase;
+
+/**
+ * @covers \WP_Rocket\ThirdParty\Hostings\Cloudways::varnish_ip
+ * @group Cloudways
+ * @group ThirdParty
+ */
+class Test_VarnishIP extends TestCase {
+	public function testShouldReturnCloudwaysVarnishIP() {
+		$varnish_ip = [];
+		$expected   = [
+			'127.0.0.1:8080',
+		];
```

Comment:
Move to test data fixture. Add more use cases such as:

- An empty array is given
- Other IPs are given
- If a non-array _could_ be given, add that